core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -800,6 +800,17 @@ private SqlLibraryOperators() {
           OperandTypes.STRING_STRING,
           SqlFunctionCategory.TIMEDATE);
 
+  /**Same as {@link #TO_TIMESTAMP}, except ,if the conversion cannot be performed,
+   * it returns a NULL value instead of raising an error.*/
+  @LibraryOperator(libraries = {SNOWFLAKE})
+  public static final SqlFunction TRY_TO_TIMESTAMP =
+      new SqlFunction("TRY_TO_TIMESTAMP",
+          SqlKind.OTHER_FUNCTION,
+          ReturnTypes.TIMESTAMP_NULLABLE,
+          null,
+          OperandTypes.STRING_STRING,
```

Comment:
should be ```
OperandTypes.or(
                            OperandTypes.STRING,
                            OperandTypes.STRING_STRING) 
```
This would make it in adherence to the behavior of try_to_timestamp in Snowflake.

Reply:
changed